docker/start.sh

```diff
@@ -64,6 +64,7 @@ CROND=$(echo "$RUN_CROND" | tr '[:upper:]' '[:lower:]')
 if [ "$CROND" = "true" -o "$CROND" = "1" ]; then
     echo "init:crond  | Cron Daemon (crond) will be run as requested by s6" 1>&2
     rm -f /app/gogs/docker/s6/crond/down
+    /bin/sh /app/gogs/docker/runtime/backup-init.sh "${PUID}"
```

Comment:
@unknwon You see, the way I call backup-init.sh? 
It will be executed in different process and exit code will not affect start.sh unless we want it to. Eg. with `set -e`
If you don't feel confident, take a look at this popular StackOverflow answer: https://stackoverflow.com/a/8352939

Another way would be to run the script in the background, asynchronously by appending ` &` to the end of the line. But that would possibly (?) affect log stream negatively. It's up to experiment.

Reply:
Gotcha, thanks for the link!